ahfe_tutorial/python_tutorial.ipynb

```diff
@@ -0,0 +1,930 @@
+{
```

Comment:
<p>And here above this one:</p><p> </p><p>"</p><p><strong>CLI / Quickrun</strong></p><p> </p><p>If you ran the simulation using the CLI (i.e. by calling <code>openfe quickrun</code> ) you will end up with the same JSON output file as the one created in the previous cell. To get your simulation results you can load them back into Python in the following manner:</p><p>"</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenFreeEnergy/ExampleNotebooks/pull/100/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='42'/>